routes/notificationRouter.js

```diff
@@ -1,8 +1,83 @@
-// const express = require('express');
-// const { db } = require('../server/db');
+const express = require('express');
+const { db } = require('../server/db');
 
-// const notificationRouter = express.Router();
+const notificationRouter = express.Router();
 
-// Your code below (delete this line when committing):
+module.exports = notificationRouter;
```

Comment:
This should be the very last line of the file. Otherwise this export might get lost with the rest of the code